casts/cards/src/UICollectionCard/types.ts

```diff
@@ -29,7 +29,11 @@ export type UICollectionCardProps = {
 };
 
 export type PreviewProps = {
-    contentList: MediaCardContent[];
+    contentList?: MediaCardContent[];
+    /**
+     * The callback is called when all content elements fail
+     */
+    onFailure: () => void;
```

Comment:
Same as for `onError` (or maybe `reasons: sting`)
```suggestion
    onFailure: (error: Error) => void;
```